provisioning/e2e/_service_create_delete.js

```diff
@@ -29,32 +28,35 @@ var enrollmentGroup = {
     x509: {
       signingCertificates: {
         primary: {
-          certificate: process.env.IOTHUB_CA_ROOT_CERT
+          certificate: new Buffer(process.env.IOTHUB_CA_ROOT_CERT, 'base64').toString('ascii')
         }
       }
     }
-  }
+  },
+  provisioningStatus: "enabled"
 };
 
-var testDescription =   [
+var testSpecification =   [
```

Comment:
>    [](start = 23, length = 3)

nit: not a fan of the formatting here - specifically: 3 spaces between the = and [ and no indentation for the objects within the array?

Reply:
Fixed.

---
In reply to: [154166762](https://github.com/Azure/azure-iot-sdk-node/pull/165#discussion_r154166762) [](ancestors = 154166762)